pom.xml

```diff
@@ -11,7 +11,7 @@
     </parent>
 
     <artifactId>workflow-graph-lib</artifactId>
-    <version>1.11.0-SNAPSHOT</version>
+    <version>1.11.0</version>
```

Comment:
Usually we leave the SNAPSHOT version in develop and update the version number when we release to `main`, are we breaking this process intentionally?